src/main/java/seedu/address/model/person/Photo.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+import java.awt.Image;
+import java.io.IOException;
+import java.net.URL;
+import javax.imageio.ImageIO;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+/**
+ * Represents a photo of a Person in the addressbook.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPhoto(String)}
+ */
+public class Photo {
+    public static final String MESSAGE_PHOTO_CONSTRAINTS =
+            "Person's photo must have a valid image URL.";
+
+    public static final String PHOTO_VALIDATION_REGEX = "[^\\s].*";
+    private static final String DEFAULT_PHOTO = "file:///C:/Users/User/Desktop/repos intelliJ/addressbook-level4" +
+            "(mine)/src/main/resources/images/defaultPhoto.png";
+
+    private Image image;
+    public String value;
+
+    /**
+     * Validates given photo
+     *
+     * @throws IllegalValueException if given photo url is invalid.
+     */
+    public Photo() {
+        value = DEFAULT_PHOTO;
+    }
+
+    public Photo(String photo) throws IllegalValueException {
+        requireNonNull(photo);
+        if (!isValidPhoto(photo)) {
+            throw new IllegalValueException(MESSAGE_PHOTO_CONSTRAINTS);
+        }
+        this.value = photo;
+    }
+
+    /**
+     * Returns true if a given string is a valid url.
+     */
+    private boolean isValidPhoto(String test) {
+        if(test.equals("images/defaultPhoto.png")){
+            return true;
+        } else if (test.matches(PHOTO_VALIDATION_REGEX)) {
```

Comment:
You may want to split it, your else if into just a single if. The arrow head indentation is slightly visible